docs/index.bs

```diff
@@ -900,46 +900,58 @@ spec: webappsec-referrer-policy; urlPrefix: https://w3c.github.io/webappsec-refe
     <section algorithm="navigation-preload-manager-enable">
       <h4 id="navigation-preload-manager-enable">{{NavigationPreloadManager/enable()}}</h4>
 
-      The <dfn method for="NavigationPreloadManager"><code>enable()</code></dfn> method, when invoked, *must* return a new [=promise=] |promise| and run the following steps [=in parallel=]:
```

Comment:
Is this shorthand out of favour now (if it ever was)?

Reply:
Yeah, the reader's eyes skip over the prelude and miss important facts like promise creation or running in parallel.